java/java.source.base/src/org/netbeans/api/java/source/TreeUtilities.java

```diff
@@ -1335,6 +1335,54 @@ public Void visitMethod(MethodTree node, Boolean p) {
         return null;
     }
     
+    /**Find the target of <code>break</code> or <code>continue</code>. The given
+     * {@link CompilationInfo} has to be at least in the {@link Phase#RESOLVED} phase.
+     * 
+     * @param breakOrContinue {@link TreePath} to the tree that should be inspected.
+     *                        The <code>breakOrContinue.getLeaf().getKind()</code>
+     *                        has to be either {@link Kind#BREAK} or {@link Kind#CONTINUE}, or
+     *                        an IllegalArgumentException is thrown
+     * @return the tree that is the "target" for the given break or continue statement, or null if there is none. Tree can be of type StatementTree or ExpressionTree
+     * @throws IllegalArgumentException if the given tree is not a break or continue tree or if the given {@link CompilationInfo}
+     *         is not in the {@link Phase#RESOLVED} phase.
+     * @since 0.16
+     */
+    public Tree getBreakContinueTargetTree(TreePath breakOrContinue) throws IllegalArgumentException {
```

Comment:
getBreakContinueTargetTree is a new API, it should have an entry in apichanges.xml and also version needs to be incremented

Reply:
+1

Reply:
Thanks for the review, Arunava. I've moved the version to 2.40.0, and made the necessary changes in TreeUtilities, and apichanges.xml.